workflows/eager.nf

```diff
@@ -351,7 +359,43 @@ workflow EAGER {
 
     }
 
-    //
+// MODULE: QUALIMAP_BAMQC
+//
+if ( !params.skip_qualimap & params.run_contamination_estimation_angsd ) {
+        snpcapture_bed_input = ch_snpcapture_bed
+            .map {
+                // Create additional map containing only meta.id for combining samples and snpcapture
+                meta, snpcapture ->
+                    meta2 = [:]
+                    meta2.reference = meta.id
+                [ meta2, meta, snpcapture ]
+            }
+        qualimap_input = ch_dedupped_bams
+            .map {
+             // Create additional map containing only meta.reference for combining samples and snpcapture
+                meta, bam, bai ->
+                    meta2 = [:]
+                    meta2.reference = meta.reference
+                [ meta2, meta, bam, bai ]
+            }
+            .combine(
+                by: 0,
+                snpcapture_bed_input
```

Comment:
See my previous comment. Since `ch_snpcapture_bed` lacks a `meta`, all this will likely create an empty channel and hence won't run the module. Once the meta is added, things should go through the module.

Reply:
Make sense. I think now meta is there